Fixes https://github.com/nebulab/solidus_pay_tomorrow/issues/23

abhishekgupta5 opened the pull request:
Fixes https://github.com/nebulab/solidus_pay_tomorrow/issues/23

**Brief:**
This PR intends to change attributes in the create_order call body based on the mentioned [issue](https://github.com/nebulab/solidus_pay_tomorrow/issues/23) -

1. Use shipping_address instead of bill_address
2. Don't convert to int for order total
3. Added taxes and shipping to the order
4. Use item's name instead of description. Since description can be too long
5. Use item's total instead of price. Price doesn't include quantity and taxes
6. Updates .gitignore

DanielePalombo reviewed (approved):
IMO more commits would have made this PR easier to review.

abhishekgupta5 committed:
Update .gitignore

Added .tool-versions for asdf local config

abhishekgupta5 committed:
Use ship_address in CreateOrderService

We should use ship_address for order creation
instead of bill_address. Since, this is shown on
the merchant's dashboard under the ship address field

Ref: https://docs.paytomorrow.com/docs/api-reference/api/create-order

abhishekgupta5 committed:
Added taxes and shipping to create_order body

Having these fields will show them on the
merchant's dasboard which is a good to have

Also removed to_i from loan_amount since
decimal is important

abhishekgupta5 committed:
Update attrubute values in applicationItems

1. Description can be too big. So use item's name instead
2. Use item.total instead of item.price because price doesn't
   consider about the quantity
3. item.sku is valid so no need for item.variant.sku
